src/App.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react'
+import Header from 'components/Header'
+import OtherProjects from 'components/OtherProjects'
+import AboutMe from 'components/AboutMe'
+import Tech from 'components/Tech'
+import FeaturedProjectCard from 'components/FeaturedProjects'
+import MyThoughts from 'components/MyThoughts'
+import Skills from 'components/Skills'
+import ForMore from 'components/ForMore'
+import Contact from 'components/Contact'
+import { GlobalFonts } from 'components/GlobalStyleComponents'
+
+/* Component that prints all the importet compnents */
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <GlobalFonts />
+      <Header />
+      <AboutMe />
+      <Tech color="#f0a04b" />
```

Comment:
Good way of solving the colour-theme in a way that can easily be overviewed and changed.